app/controllers/concerns/authenticated_state_file_intake_concern.rb

```diff
@@ -0,0 +1,16 @@
+module AuthenticatedStateFileIntakeConcern
+  extend ActiveSupport::Concern
+
+  included do
+    before_action :require_state_file_intake_login
+  end
+
+  private
+
+  def require_state_file_intake_login
+    if current_state_file_az_intake.blank? && current_state_file_ny_intake.blank?
```

Comment:
I wonder if we can make one clear method that we can use, but I'm not sure exactly what it would be.